ci/variables.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -exuo pipefail
+
+readonly amd64_arch=x86_64
+readonly arm_arch=arm32v7
+readonly arm64_arch=arm64v8
+readonly default_repo_url=public.ecr.aws/sumologic/fluent-bit
+
+REPO_URL=${REPO_URL:-$default_repo_url}
+
+DOT="$(dirname "$(realpath "${BASH_SOURCE[0]}")")"
```

Comment:
What does mean DOT? Could we use more meaningful name?

Reply:
I spent some time trying to come up with a name that would be both descriptive and short/readable. Apparently I failed 😄 This variable means current directory of the executing script. With the `$DOT` I was going for `$DOT/script.sh` being more-or-less equivalent to `./script.sh`.

Reply:
`CI_ROOT`, `CI_DIR` maybe?